services/upload/src/main/resources/application.yaml

```diff
@@ -25,6 +25,8 @@ services:
     min-batch-key-count: 140
     # The maximum number of keys that an upload batch can contain
     max-batch-key-count: 4000
+    # Time window Federation Upload Keys will be loaded from DB
+    retention-days: 14
```

Comment:
7

Reply:
I think if we have anything smaller than 14 we risk losing keys, wouldn't we? 
The filter here is by rolling start interval - this means in a single submission we can have keys with rolling start interval from 14 days back if I'm not mistaken.